README.md

```diff
@@ -12,7 +12,7 @@ This Puppet module provides secure configuration of your base OS with hardening.
 
 * Puppet OpenSource or Enterprise
 * [Module stdlib](https://forge.puppet.com/puppetlabs/stdlib)
-* [Module sysctl](https://forge.puppet.com/thias/sysctl)
+* [Module sysctl](https://github.com/hercules-team/augeasproviders_sysctl)
```

Comment:
I think we should keep this consistent and link here to the puppetforge entry (https://forge.puppet.com/herculesteam/augeasproviders_sysctl)